.github/workflows/publish-docs.yaml

```diff
@@ -21,42 +41,54 @@ jobs:
         branch: release
         name: docs-stable
         path: docs-release
+        if_no_artifact_found: ignore
 
-    - name: 📥 Download latest docs
+    - name: 📥 Download latest unstable docs
       uses: dawidd6/action-download-artifact@v2
       with:
         workflow: ci.yaml
         workflow_conclusion: success
         branch: master
-        name: docs-stable
-        path: docs-master
+        name: docs
+        path: docs-unstable
+        # TODO: backward compatibility
+        # remove the following line to throw an error instead
+        # of a warning if artifact is not found when we have
+        # a release branch with artifact name docs instead of
+        # docs-stable
+        if_no_artifact_found: warn
 
-    - name: 📥 Download latest /unstable docs
+    # TODO: backward compatibility
+    # remove the following block when we have
+    # a release branch with artifact name docs instead of
+    # docs-stable
+    - name: 📥 Download latest unstable docs
       uses: dawidd6/action-download-artifact@v2
       with:
         workflow: ci.yaml
         workflow_conclusion: success
         branch: master
-        name: docs-unstable
-        path: docs-master-unstable
+        name: docs-stable
```

Comment:
The docs-stable artifact from master builds would include non-modified `baseUrl` and should not be used as an "unstable" site.

In general, why would we need this? The moment this workflow runs after the modified `ci` run for branch `master`, we would have a `docs` artifact we can use here?

I think we can just the above job and remove this one right away.

Reply:
Nice catch 🤯 @ch1bo 

Indeed, next time master runs, we will have access to a _well known/named_ artifact. Let's get rid of this TODO already!

Thanks.